src/components/Modals/CreateAssetPackModal/CreateAssetPackModal.tsx

```diff
@@ -23,12 +23,24 @@ export default class CreateAssetPackModal extends React.PureComponent<Props, Sta
 
   analytics = getAnalytics()
 
+  getDefaultAssetPackName() {
+    const { assetPacks } = this.props
+    const defaultName = t('asset_pack.default_name')
+    let name = defaultName
+    let suffix = 2
+    while (assetPacks.some(assetPack => assetPack.title === name)) {
+      name = `${defaultName} (${suffix})`
+      suffix++
+    }
+    return name
```

Comment:
Suggestion: you could add the suffix as a variable in the translation, to avoid problems if we add languages that read from right to left. Like "default_name": "My Asset Pack ({suffix})"
and then
`return t('asset_pack.default_name', { suffix })`

Not really necessary now tho